abb_irb120t_moveit_config/package.xml

```diff
@@ -1,39 +1,32 @@
-<?xml version="1.0"?>
-<package format="2">
+<package>
+
   <name>abb_irb120t_moveit_config</name>
-  <version>0.2.0</version>
+  <version>0.3.0</version>
   <description>
-    <p>
-      MoveIt package for the ABB IRB 120T.
-    </p>
-    <p>
-      An automatically generated package with all the configuration and launch
-      files for using the ABB IRB 120T with the MoveIt Motion Planning
-      Framework.
-    </p>
+     An automatically generated package with all the configuration and launch files for using the abb_irb120t_3_58 with the MoveIt! Motion Planning Framework
   </description>
-  <author email="assistant@moveit.ros.org">MoveIt Setup Assistant</author>
+  <author email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</author>
   <maintainer email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</maintainer>
 
   <license>BSD</license>
 
   <url type="website">http://moveit.ros.org/</url>
-  <url type="bugtracker">https://github.com/ros-industrial/abb_experimental/issues</url>
-  <url type="repository">https://github.com/ros-industrial/abb_experimental</url>
+  <url type="bugtracker">https://github.com/ros-planning/moveit/issues</url>
+  <url type="repository">https://github.com/ros-planning/moveit</url>
```

Comment:
@Jmeyer1292: easiest would probably to revert the changes to `package.xml`. The new MSA doesn't add anything beyond what was already in the manifest.